### DISCUSSION TITLE
How do I see Flet app with Mobile screen size on Windows?

### DISCUSSION THREAD

**Discussion by basatsu** [flet-dev/flet#1138] [Q&A]
### Question

I want to watch Flet mobile app with Mobile screen size on Windows.

This is what I am expecting.
You run the below command  
`flet run [filename] -d`
Then the app is hosted on your local.
I want to watch Flet mobile app on Windows here.

I know we can see the app on Windows with this command
`flet.app(target=main, view=flet.WEB_BROWSER)`
but I wonder if it is possible to see the app with Mobile screen size on WIndows.
Do you have any ideas?  

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by saurabhwadekar** [comment]
I think you should wait for Flet Studio to come
